### DISCUSSION TITLE
Use BullMQJobQueueStrategy

### DISCUSSION THREAD

**Discussion by Hunglmc** [vendurehq/vendure#2856] [Q&A]
Hello, I'm learning and using BullMQJobQueueStrategy, but I can't find a way to initiate a job processing flow. My problem is that batch approval with id will retrieve all foreign key records belongs to the main id, then make a test api call, after the api checks the response, continue to post and update the record status in the original database, each record makes two api calls. Can you give me instructions with this library

**Comment by michaelbromley** [comment]
Hi,

I don't fully understand what is being done here. Could you please post the actual API calls you are making and list the outputs/errors you encounter?
